### ISSUE TITLE
LeNet with Conv2 + FC1 + FC2

### ISSUE THREAD

**Issue by tung-qle** [RuiLin0212/DeBut#1]
Hi,
Thanks for the implementation and the paper. I'm testing your code on LeNet Conv2 + FC1 + FC2 Debut with ALS initialization. However, I cannot get better than 94% accuracy (with ALS initialization). Other variants work as described in the paper.
Can you please give me a bash file that runs all the commands for the case LeNet Conv2 + FC1 + FC2 Debut with ALS initialization (from initialization to training step)?
